source/agora/node/FullNode.d

```diff
@@ -466,4 +495,65 @@ public class FullNode : API
     protected void onAcceptedTransaction () @safe
     {
     }
+
+    /***************************************************************************
+
+        Push the block data to the `block_handlers` target server list
+        set in config.
+
+        Convert block data to JSON serialization and send it POST using Rest.
+
+        Params:
+            block = externalized block data
+
+    ***************************************************************************/
+
+    private void pushBlock (const ref Block block) @safe
+    {
+        foreach (address, handler; this.block_handlers)
+        {
+            runTask({
+                try
+                {
+                    handler.pushBlock(block);
```

Comment:
I have a question.
Will this value(block) be valid here?

https://github.com/bpfkorea/agora/blob/8056af52285f23eb6937102c3bd7662c013eec2c/source/agora/consensus/protocol/Nominator.d#L395

Reply:
I'm not sure, 
I think he's asking if the block is consensused block. Is this what you meant?
And do you want to know if here the `pushBlock` call point is okay?

Reply:
`pushBlock` is called when a **validated** block is added to the ledger. It is unrelated to nomination.

Reply:
@AndrejMitrovic 
When a local variable created by another function is approached within the runTask,
The local variable's contents are missing.
How should I solve this?
I upload the sample code I made together.
Please.
https://gist.github.com/MichaelKim20/0c626a54dfd1f68593fb886da059c20b

Reply:
Ah that's right, we should not use `ref` here. The reference may be out of scope when the task starts.

@TrustHenry you will need to remove `ref` from the function parameter.

Reply:
Yes, it certainly does.
Thank you.
I will apply `pushPreImage` as well.